sycl/include/CL/sycl/detail/pi.h

```diff
@@ -1823,9 +1823,9 @@ struct _pi_plugin {
   // Some choices are:
   // - Use of integers to keep major and minor version.
   // - Keeping char* Versions.
-  char PiVersion[5];
+  char PiVersion[10];
```

Comment:
Let's make this size into a "const int" parameter, and stop needing to sync all users.

Reply:
Some backends simply use `sizeof(PluginVersion)`. I have propagated this to backends without it.